src/app/MessageDef/ReadRequestMessage.h

```diff
@@ -34,11 +35,12 @@ namespace app {
 namespace ReadRequestMessage {
 enum class Tag : uint8_t
 {
-    kAttributeRequests  = 0,
-    kDataVersionFilters = 1,
-    kEventRequests      = 2,
-    kEventFilters       = 3,
-    kIsFabricFiltered   = 4,
+    kAttributeRequests        = 0,
+    kDataVersionFilters       = 1,
+    kEventRequests            = 2,
+    kEventFilters             = 3,
+    kIsFabricFiltered         = 4,
+    kInteractionModelRevision = 5,
```

Comment:
Didn't we agree to have the tag value for this be at the end of the tag space? (i.e 0xFF)

More-over, I'd rather such a tag be defined once in a central place, and referenced by all.

Reply:
yes, update it with 0xFF, yes, update it using central place(MessageParser, MessageBuilder) to hold shared logic, thanks